iso15118/secc/controller/simulator.py

```diff
@@ -908,3 +953,15 @@ async def get_15118_ev_certificate(
         ).decode("utf-8")
 
         return base64_encode_cert_install_res
+
+    def get_cfg_parameter(self, string, default):
+        try:
+                if self.config_param[string] is None:                      
+                        raise Exception("revert to default")
```

Comment:
there is no point on raising a general exception here. log as info and return the default

Reply:
Updated! Thanks for the suggestion